From 54500283b0a47b73fc0e6c68d9eb12c91c3a7fc5 Mon Sep 17 00:00:00 2001
From: Ady Abraham <adyabr@google.com>
Date: Sat, 23 Nov 2024 02:52:44 +0000
Subject: [PATCH 7/7] SF: VsyncTimeline::isVSyncInPhase should use display rate
 [unflagged]

Remove the flag for the fix in commit 4a719e88f8050713cc75d655f3e75473e5404327 so it will be enabled on android15-tests-dev (GSI) release.

Bug: 328352850
Bug: 355684882
Bug: 380359880
Test: see b/355684882
Flags: EXEMPT bugfix
Change-Id: I02deca58bfd9a2cf169c2e58995cc42eeb854c8d
Merged-In: I02deca58bfd9a2cf169c2e58995cc42eeb854c8d
---
 services/surfaceflinger/Scheduler/VSyncPredictor.cpp | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/services/surfaceflinger/Scheduler/VSyncPredictor.cpp b/services/surfaceflinger/Scheduler/VSyncPredictor.cpp
index ff360b754c..2d7f26e0f5 100644
--- a/services/surfaceflinger/Scheduler/VSyncPredictor.cpp
+++ b/services/surfaceflinger/Scheduler/VSyncPredictor.cpp
@@ -780,9 +780,7 @@ bool VSyncPredictor::VsyncTimeline::isVSyncInPhase(Model model, nsecs_t vsync, F
         return ticks<std::milli, float>(TimePoint::fromNs(timePoint) - now);
     };
 
-    Fps displayFps = !FlagManager::getInstance().vrr_bugfix_24q4() && mRenderRateOpt
-            ? *mRenderRateOpt
-            : Fps::fromPeriodNsecs(mIdealPeriod.ns());
+    Fps displayFps = Fps::fromPeriodNsecs(mIdealPeriod.ns());
     const auto divisor = RefreshRateSelector::getFrameRateDivisor(displayFps, frameRate);
     const auto now = TimePoint::now();
 
-- 
2.48.1

